### PR TITLE
Accountmanager doctrine mysql come back"

### PR DESCRIPTION
**Pull request by wirwolf**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "facile-it/doctrine-mysql-come-back",
+    "name": "accountmanager/doctrine-mysql-come-back",
```
Comment (peelandsee) [inline composer.json]: ahahah nice try

File: src/Driver/ServerGoneAwayExceptionsAwareTrait.php [hunk 2]
```diff
@@ -11,11 +11,15 @@ trait ServerGoneAwayExceptionsAwareTrait
     protected $goneAwayExceptions = [
         'MySQL server has gone away',
         'Lost connection to MySQL server during query',
+        'The MySQL server is running with the --read-only option so it cannot execute this statement',
+        'Connection refused'
```
Comment (peelandsee) [inline src/Driver/ServerGoneAwayExceptionsAwareTrait.php]: not a "mysql has gone away" error type..

File: src/Driver/ServerGoneAwayExceptionsAwareTrait.php [hunk 3]
```diff
@@ -11,11 +11,15 @@ trait ServerGoneAwayExceptionsAwareTrait
...
     ];
 
     /** @var string[] */
     protected $goneAwayInUpdateExceptions = [
         'MySQL server has gone away',
+        'The MySQL server is running with the --read-only option so it cannot execute this statement',
```
Comment (peelandsee) [inline src/Driver/ServerGoneAwayExceptionsAwareTrait.php]: not a "mysql has gone away" error type..

**Comment by wirwolf** [comment]
I'm sorry. I wanted to do this in my repository

**Pull request closed without merging**